notebooks/regression/tutorial_regression.md

```diff
@@ -108,6 +108,16 @@ polyn_model = Pipeline(
         ("linear", LinearRegression())
     ]
 )
+polyn_model_quant = Pipeline(
+    [
+        ("poly", PolynomialFeatures(degree=degree_polyn)),
+        ("linear", QuantileRegressor(
+                solver="highs-ds",
+                alpha=0.01,
+                fit_intercept=False
```

Comment:
Does `True` or `False` change something to the results ?

Reply:
On personal experience, it would seem that `fit_intercept=False` seems to give better results, however, on the `QuantileRegressor` class it is set by default to `True`. So I will just keep it at that for the moment, simply using `QuantileRegressor` without setting `fit_intercept` to any value.